src/chainparams.cpp

```diff
@@ -211,7 +211,7 @@ class CTestNetParams : public CChainParams {
 
         // The best chain should have at least this much work.
         consensus.nMinimumChainWork = uint256S(
-            "0x00");
+            "0000000000000000000000000000000000000000000000416ad051088ff75074");
 
```

Comment:
Think we should keep this at 0

Reply:
Look at the files changes and not the commits on this one pls. I got mixed up this AM and reverted a commit I didn't mean to. It's fixed though